drv/sidecar-front-io/sidecar_qsfp_x32_controller.json

```diff
@@ -322,6 +322,17 @@
                             "se_onread": null,
                             "se_onwrite": null,
                             "desc": "The VSC8562 has been brought out of reset and is ready for\nsoftware control"
+                        },
+                        {
+                            "type": "field",
+                            "inst_name": "PG_TIMED_OUT",
+                            "lsb": 7,
+                            "msb": 7,
+                            "reset": 0,
+                            "sw_access": "r",
+                            "se_onread": null,
+                            "se_onwrite": null,
+                            "desc": "Either the V1P0 or V2P5 rail has failed to come up. V1P0 is\nfirst in the sequence, its PG is the EN for V2P5."
```

Comment:
Related to the question above – it's weird to have hard linebreaks in the JSON file, since they're not in sensible places.